core/controllers/base.py

```diff
@@ -367,7 +367,8 @@ def _render_exception_json_or_html(self, return_type, values):
                     'error-iframed.mainpage.html',
                     iframe_restriction=None)
             else:
-                self.render_template('error-page.mainpage.html')
+                self.render_template(
+                    'error-page-%s.mainpage.html' % values['status_code'])
```

Comment:
Can we be sure that `values['status_code']` will only be one of the values that we have error page create for (400, 401, 404, 500)?

Reply:
From what I see in base.py https://github.com/oppia/oppia/blob/05094e9226d599f662f5e92842346b3763aa991a/core/controllers/base.py#L386

Reply:
Okay, then please add a comment there that would say that if someone is adding new code there they should also create the HTML error page for that code.